src/general/OptionsMenu.cs

```diff
@@ -791,4 +813,12 @@ private void OnMaxQuickSavesValueChanged(float value)
 
         CompareSettings();
     }
+
+    private void OnLanguageSettingSelected(int item)
```

Comment:
I think there should be a way to go back to the default language.
If nothing else there should be like a button to set the language to null, and restart the game, to get the default language back.

Reply:
There is a reset to default button already for settings in general, isn't that enough?

Reply:
It isn't because what if you just want to have the default language option and don't want to reset everything?

Reply:
Then you can just select it in the list, I fail to see where this would be an issue.

Reply:
Because the default means "Godot selects the language based on the user's computer (and C# selects number format for the user's language)"
and explicitly selected means: "Use this language always"